harness/testTypedArray.js

```diff
@@ -120,3 +121,27 @@ function testTypedArrayConversions(byteConversionValues, fn) {
     });
   });
 }
+
+function createTypedArrayVariations(TA, values) {
+  const rab = new ArrayBuffer(4 * TA.BYTES_PER_ELEMENT, { maxByteLength: 8 * TA.BYTES_PER_ELEMENT });
```

Comment:
Why only ArrayBuffer; what's the plan for SharedArrayBuffer? Should those TAs be created by this function or not?